zproject/default_settings.py

```diff
@@ -98,7 +98,8 @@
 SOCIAL_AUTH_APPLE_EMAIL_AS_USERNAME = True
 
 # Generic OpenID Connect:
-SOCIAL_AUTH_GENERIC_OIDC_NAME = ""
+SOCIAL_AUTH_OIDC_ENABLED_IDPS: Dict[str, Dict[str, Optional[str]]] = {"unconfigured": {}}
```

Comment:
This should just be an empty dict, right?

Reply:
I see, you're checking for exactly 1 above -- is that why we need this?  Might be easier to just do `<=1`.

Reply:
It was to make the awkward line in backends.py not throw an exception - but I tweaked the line to:
```
settings_dict = list(settings.SOCIAL_AUTH_OIDC_ENABLED_IDPS.values() or [{}])[0]
```

and now the dict can be empty. The line is very ugly though :disappointed: